keps/sig-testing/2420-reducing-kubernetes-build-maintenance/README.md

```diff
@@ -380,9 +380,6 @@ n/a.
 
 ## Production Readiness Review Questionnaire
 
-**TODO**: This entire section seems completely irrelevant for KEPs that do not
```

Comment:
separate follow up re: #2311, this should probably be limited to keps that do target production, as opposed to say, test tools or meta procedural stuff like the KEP KEP, saving everyone time in that case ...